### PR TITLE
update ee user management docs

### PR DESCRIPTION
**Pull request by pwilson77**

### CODE REVIEW

File: pro/pro-users.rst [hunk 1]
```diff
@@ -1,194 +1,55 @@
...
-The Star ("*") Group
--------------------------
-
-By default, every User is added to the * group. By default, every network has * in allowed groups. This means that if you create a new network, all users will have the "Default Access Level" and limits.
-
-To limit a network to particular users, remove the * group from Allowed Groups.
+* User: A user does not have access to the Netmaker dashboard. They can only use the remote access client to connect to a gateway.
 
-To limit a user to particular networks, remove the * group from their group settings.
+* Admin: An admin has full access to the Netmaker dashboard. They can create and manage users, networks, and gateways. They can also manage user permissions.
```
Comment (Aceix) [inline pro/pro-users.rst:10]: missing notes on superuser

File: pro/pro-users.rst [hunk 2]
```diff
@@ -1,194 +1,55 @@
...
+   :alt: Add User
    :align: center
 
+The credentials will need to be shared with the added user.
 
-Level 1: Node Access 4
-------------------------
-
-Now let's modify this user's Access Level to 1. Now, when they log in, they will have two options in their panel. Remote Access, and Nodes.
+Attaching or removing user from ingress
```
Comment (Aceix) [inline pro/pro-users.rst:27]: let's say "Client Gateway" instead of "Ingress'

File: pro/pro-users.rst [hunk 3]
```diff
@@ -1,194 +1,55 @@
...
+=================================
+To attach users to a gateway ore remove users from a gateway, you will need to have the gateway set up.
```
Comment (Aceix) [inline pro/pro-users.rst:29]: typo "or" not "ore"

File: pro/images/dashboard-no-access.png
Comment (Aceix) [inline pro/images/dashboard-no-access.png:1]: this page is not in the new ui. we shouldnt add it

**Review by Aceix — CHANGES_REQUESTED** [review]

File: pro/pro-users.rst
Comment (abhishek9686) [inline pro/pro-users.rst:1]: @pwilson77  also mention that only superadmin and admins can visit the dashboard
Touch Upon the normal users with gateways and users functionality with remote access client

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: pro/pro-users.rst [hunk 4]
```diff
@@ -1,194 +1,82 @@
...
-.. image:: images/users/user-dashboard-1.png
-   :width: 80%
-   :alt: Select Network
-   :align: center
+The credentials will need to be shared with the added user.
 
-They will then be at their Dashboard, and will need to select an option. Since they are Level 2, they have only one option, "Remote Access:"
+Attaching or removing user from a client gateway
+================================================
+To attach users to a gateway or remove users from a gateway, you will need to have the gateway set up.
```
Comment (abhishek9686) [inline pro/pro-users.rst:30]: gateway -> client gateway

File: pro/pro-users.rst [hunk 5]
```diff
@@ -1,194 +1,82 @@
...
-   :width: 80%
-   :alt: Select Network
-   :align: center
+The credentials will need to be shared with the added user.
 
-They will then be at their Dashboard, and will need to select an option. Since they are Level 2, they have only one option, "Remote Access:"
+Attaching or removing user from a client gateway
+================================================
+To attach users to a gateway or remove users from a gateway, you will need to have the gateway set up. 
+Once the gateway is set up, you will see an option to remove users from the dropdown menu.
```
Comment (abhishek9686) [inline pro/pro-users.rst:31]: gateway -> client gateway

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: pro/pro-users.rst [hunk 6]
```diff
@@ -1,194 +1,82 @@
...
 
-.. image:: images/users/user-dashboard-4.png
-   :width: 80%
-   :alt: Remote Access 3
-   :align: center
+Transferring super admin rights
+===============================
 
+Super admin rights can be transferred only to another admin. To do this, on the users page, go to the superadmin row and hover over the ellipsis.
+In the dropdown menu, an option to transfer admin rights should be present. On clicking it, a dialog box will open allowing you to select any admin
```
Comment (abhishek9686) [inline pro/pro-users.rst:50]: `In the dropdown menu, an option to transfer admin rights should be present` -> you will see an option to transfer admin rights`

File: pro/pro-users.rst [hunk 7]
```diff
@@ -1,194 +1,82 @@
...
    :width: 80%
-   :alt: Node User 2
+   :alt: Remote access login
    :align: center
 
-.. image:: images/users/user-dashboard-8.png
-   :width: 80%
-   :alt: Node User 3
-   :align: center
+Then they will have to select a network and subsequently a client gateway to use and after click on connect
```
Comment (abhishek9686) [inline pro/pro-users.rst:67]: `Then they will have to select a network and subsequently a client gateway to use and after click on connect`. -> `After successful login you will be shown all the networks and gateways you have given access to, so now you will be able to connect/disconnect/refresh your connection to a gateway`

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: pro/pro-users.rst (hunk 5 above)
Comment (pwilson77) [reply pro/pro-users.rst]: done

**Review by Aceix — APPROVED** [review]

**Review by abhishek9686 — APPROVED** [review]

**Pull request merged**